ansible/playbooks/roles/containerd/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Remove Docker
```

Comment:
How about stopping `kubelet` before removing Docker? Haven't checked but I suspect it can produce many errors in logs after stopping Docker.